### ISSUE TITLE
Issues and ideas re. protein dimerization/oligomerization terms

### ISSUE THREAD

**Issue by paolaroncaglia** [geneontology/go-ontology#13036]
This ticket is meant as a reference point to summarise concerns about protein dimerization activity + protein oligomerization terms, and to collect links to previous discussions and proposals. Hopefully this will aid towards resolution of a long-standing issue within the GOC that affects many annotations.

In chronological order (most recent last):

Old minutes/links:
http://wiki.geneontology.org/index.php/Annotation_Advocacy_progress_report_for_2013
http://wiki.geneontology.org/index.php/Annotation_Conf._Call_June_11,_2013
http://wiki.geneontology.org/index.php/Annotation_Conf._Call_July_9,_2013
http://wiki.geneontology.org/index.php/Annotation_consistency:_Clarification_of_oligomerization,_dimerization,_protein_complex_assembly

Previous GH tickets:
https://github.com/geneontology/go-ontology/issues/11094 (dimerization MF terms)
https://github.com/geneontology/go-ontology/issues/11269 (locations of "protein homodimerization activity" vs "protein homotetramerization)
https://github.com/geneontology/go-ontology/issues/11748 (please clarify what should be annotated to "protein polymerization" "microtubule polymerization" etc)

Minutes of GOC meeting in Barcelona: (search for “Protein dimerization/oligomerization”)
https://docs.google.com/document/d/1NonH97s8xEpDdx6DfonKPKI_RdHbbG-yft85UZtUmF0/edit#

Proposals sent to the GOC on Nov 6th and 24th 2014:
Go to https://mailman.stanford.edu/pipermail/go-consortium/2014-November/thread.html#start
and scroll towards the bottom to the thread [go] Protein oligomerization

Ontology editors discussion:
http://wiki.geneontology.org/index.php/Ontology_meeting_2014-12-16#Follow-up:_Protein_oligomerization

Related, recent discussion on annotating to assembly terms:
https://github.com/geneontology/go-ontology/issues/12962#issuecomment-275370449
https://github.com/geneontology/go-ontology/issues/12962#issuecomment-275374459
https://github.com/geneontology/go-ontology/issues/12962#issuecomment-275374590

Recent suggestions from @RLovering and @dosumis:
https://github.com/geneontology/go-ontology/issues/11269#issuecomment-280264317
https://github.com/geneontology/go-ontology/issues/11269#issuecomment-280365248

**Comment by RLovering** [comment]
Thanks Paola

**Comment by dosumis** [comment]
CC @pdthomas - Would be good to have your take on adding oligomerization as an MF + relationship to protein binding - bridging.

**Comment by jrr-cpt** [comment]
Related to this issue, it would be extremely helpful if the definition for the protein oligomerization were updated to reflect the intended use. To a new user, the current definition does not indicate that protein association with itself (spontaneous) is either intended or excluded. Reading through many of the comments here suggest, however, that only proteins that aid in assembly of dimers/oligomers, but are not themselves in the complex apply.
 
@jimhu-tamu @sandyl27

**Comment by pgaudet** [comment]
These terms have been used very inconsistently. 

The proposal is to obsolete these terms and move annotations to 'protein-containing complex (GO:0032991)' OR to protein-containing complex assembly (GO:0065003).

Would that be OK for you ? 

Thanks, Pascale

**Comment by jrr-cpt** [comment]
Yes, that would be fine. The only thing that would still appear to be missing is a clear presentation of whether self-association fits within the definition or not.

@jimhu-tamu
@sandyl27
